src/bp/sdk/botpress.d.ts

```diff
@@ -1185,16 +1185,18 @@ declare module 'botpress/sdk' {
     evaluate: (event: IO.IncomingEvent, params: any) => number
   }
 
-  export interface ConditionParams {
-    [paramName: string]: {
-      label: string
-      /** Each type provides a different kind of editor */
-      type: 'string' | 'number' | 'boolean' | 'list'
-      required?: boolean
-      defaultValue?: any
-      /** When type is list, this variable must be configured */
-      list?: ConditionListOptions
-    }
+  export interface ConditionParam {
```

Comment:
this is pretty non conventional, maybe using json schema like we do for content would be better ? Json schema renderer is a pain as well I know ... I'm not sure what are our options here :|

Reply:
Not enough flexibility and it adds a layer of complexity, if we want users to use them as much as possible, it should be easy to handle. 

Content are easy to manage that way, but if we want to use json schema, it's a lot of custom components to write